src/main/java/duke/Task/Deadline.java

```diff
@@ -0,0 +1,80 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task with a deadline.
+ */
+public class Deadline extends Task {
+
+    protected String by;
+
+    private final LocalDate date;
+    private final LocalTime time;
+
+    public static final String delimiterBy = " /by ";
```

Comment:
Would you name all your static variables in capital letter?

Reply:
Oh, nice catch! Thanks for pointing it out!